### PR TITLE
fix: missing headers and cookies on send request

### PR DESCRIPTION
**Pull request by marclave**
before we couldnt add headers or cookies to requests, we also break up the body appropriately now

### CODE REVIEW

File: packages/client-app/src/libs/sendRequest.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import type {
+  RequestInstance,
+  RequestInstanceParameter,
+  RequestRef,
+  ResponseInstance,
+} from '@scalar/oas-utils/entities/workspace/spec'
+import type { AxiosError, AxiosRequestConfig } from 'axios'
+import axios from 'axios'
+
+const paramsReducer = (params: RequestInstanceParameter[] = []) =>
```
Comment (amritk) [inline packages/client-app/src/libs/sendRequest.ts:10]: would just add a short jsdoc comment here

**Review by amritk — APPROVED** [review]

File: packages/client-app/src/libs/sendRequest.ts (hunk 1 above)
Comment (marclave) [reply packages/client-app/src/libs/sendRequest.ts:10]: oh yeah let me add some comments : )

Comment (marclave) [reply packages/client-app/src/libs/sendRequest.ts:15]: done

File: packages/client-app/src/libs/sendRequest.ts [hunk 2]
```diff
@@ -7,6 +7,11 @@ import type {
 import type { AxiosError, AxiosRequestConfig } from 'axios'
 import axios from 'axios'
 
+/**
+ * Convert the parameters to an object
+ * @param RequestInstanceParameter
+ * @returns Record<string, string>
```
Comment (amritk) [inline packages/client-app/src/libs/sendRequest.ts:13]: we actually don't need these when using typescript! Only useful if you want to actually describe the parameter, just the type can be inferred from typescript

Comment (marclave) [reply packages/client-app/src/libs/sendRequest.ts:13]: removed we are so back

**Pull request merged**